todo_app/data/mongo_items.py

```diff
@@ -28,3 +28,29 @@ def add_card(title):
     card = {"name": title, "state": "To Do"}
     COLLECTION.insert_one(card)
     return 0
+
+def get_myrole(ClientID):
+    #Authorised Writers
+    if ClientID.strip() in {'65459782','1'}:
+        return ""
+    # default readers
+    else:
+        return "disabled"
+
+def get_currentuser():
```

Comment:
Nice way of wrapping this to solve the issue!

Reply:
Thanks